src/Microsoft.DotNet.Cli.Utils/CsvFormatter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class CsvFormatter
+    {
+        internal static string EscapeRow(IEnumerable<string> values)
+        {
+            return values
+                .Select(EscapeValue)
+                .Aggregate((a, v) => a + "," + v);
+        }
+
+        internal static string EscapeValue(string value)
+        {
+            return "\"" + value.Replace("\\", "\\\\").Replace("\"", "\\\"") + "\"";
```

Comment:
FYI, this isn't how CSVs are conventionally escaped, as semi-codified in RFC 4180.

Backslashes should not be escaped.  Each double quote should be replaced with two double quotes:
```csharp
return "\"" + value.Replace("\"", "\"\"") + "\"";
```